cif_pow.dic

```diff
@@ -46,91 +46,247 @@ save_PD_GROUP
+        in two diffractograms, DIFFRACTOGRAM_A and DIFFRACTOGRAM_B. To find the
+        exact peak position, width and intensity details, you must find the
+        peaks indexed with the given peak id and diffractogram id combinations.
+;
 
 save_
 
-save_pd_amorphous.peak_id
+save_pd_amorphous_peak.d_spacing
 
-    _definition.id                '_pd_amorphous.peak_id'
-    _definition.update            2023-01-08
+    _definition.id                '_pd_amorphous_peak.d_spacing'
+    _definition.update            2023-06-18
     _description.text
 ;
-    An arbitrary code is assigned to each peak in an amorphous phase.
+    Peak position as a d-spacing in angstroms.
+;
+    _name.category_id             pd_amorphous_peak
+    _name.object_id               d_spacing
+    _type.purpose                 Measurand
+    _type.source                  Derived
+    _type.container               Single
+    _type.contents                Real
+    _enumeration.range            0.0:
+    _units.code                   angstroms
 
-    Used to link with _pd_peak.id. Each peak will have a unique code. In cases
-    where two peaks are severely overlapped, it may be desirable to list them as
-    a single peak.
+save_
 
-    A peak ID must be included for every amorphous peak.
+save_pd_amorphous_peak.d_spacing_su
+
+    _definition.id                '_pd_amorphous_peak.d_spacing_su'
+    _definition.update            2023-06-18
+    _description.text
 ;
-    _name.category_id             pd_amorphous
-    _name.object_id               peak_id
-    _name.linked_item_id          '_pd_peak.id'
-    _type.purpose                 Link
+    Standard uncertainty of _pd_amorphous_peak.d_spacing.
+;
+    _name.category_id             pd_amorphous_peak
+    _name.object_id               d_spacing_su
+    _name.linked_item_id          '_pd_amorphous_peak.d_spacing'
+    _units.code                   angstroms
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+
+save_
+
+save_pd_amorphous_peak.diffractogram_ids
+
+    _definition.id                '_pd_amorphous_peak.diffractogram_ids'
+    _definition.update            2023-06-18
+    _description.text
+;
+    List of diffractogram id values (_pd_diffractogram.id) associated with this
+    amorphous peak.
+
+    Must be give in the same order as _pd_amorphous_peak.peak_ids.
+    That is, the n^th^ value in _pd_amorphous_peak.peak_ids and the n^th^ value
+    in _pd_amorphous_peak.diffractogram_ids must correctly identify the peak
+    associated with this amorphous peak.
+;
+    _name.category_id             pd_amorphous_peak
+    _name.object_id               diffractogram_ids
+    _type.purpose                 Encode
     _type.source                  Assigned
-    _type.container               Single
-    _type.contents                Code
+    _type.container               List
+    _type.dimension               '[]'
+    _type.contents                Text
```

Comment:
```suggestion
    _type.contents                ByReference
    _type.contents_referenced_id  '_pd_diffractogram.id'
```

Reply:
This constructs simply means "Take the content type from `_pd_diffractogram.id`". Helps to keep the content types in sync.